### ISSUE TITLE
Set Up A Security Policy 

### ISSUE THREAD

**Issue by NotMoni** [node-fetch/node-fetch#813]
Node-fetch is used by a lot of people. 1.9million. Shouldn't we set up a Security Policy just in case someone finds a critical security bug or something along the lines of that?

Just a suggestion. 

 ![image](https://user-images.githubusercontent.com/40552237/82560082-bc70ad00-9b3e-11ea-812b-a6c879591105.png)

**Comment by xxczaki** [comment]
If we use Tidelift, we can use their [security contact](https://tidelift.com/security).

[Example](https://github.com/phpmd/phpmd/blob/master/.github/SECURITY.md)

**Comment by NotMoni** [comment]
Alright, see wait and see what others think before we make a move on this

**Comment by NotMoni** [comment]
@bitinn What do you think about this?

**Comment by bitinn** [comment]
@NotMoni I thought the last security report was handled quite gracefully, might want to ask @xxczaki @Richienb about that workflow

**Comment by Richienb** [comment]
It might make sense to use Tidelift, as @xxczaki suggested earlier

**Comment by NotMoni** [comment]
👍🏼  Sounds good.

**Comment by xxczaki** [comment]
@bitinn @Richienb I agree with Richie - I will contact Tidelift regarding this and write back on our private conversation (core team).

**Comment by NotMoni** [comment]
Sounds good!

**Comment by xxczaki** [comment]
Reopening so that we don't forget about this :smile:
